src/main/java/jdwp/Translator.java

```diff
@@ -192,15 +215,21 @@ public static String normalizeFunc(String func) {
 				newParams.add(param);
 			}
 		}
-		String newParamList = "";
-		for (int i = 0; i < newParams.size(); i++) {
-			if (i == 0) {
-				newParamList += newParams.get(i);
-			} else {
-				newParamList += ", " + newParams.get(i);
-			}
+
+		StringBuilder newParamList = new StringBuilder();
+		for (String newParam : newParams) {
+			newParamList.append(newParam);
```

Comment:
Will this add the `", "` in between the params?

Reply:
This will not because JNI parameters are not separated by commas. 

For example, if the method is: 

``` public static void downLoadProfileImage(String url, String fileName,  int tag, int from)```

then the signature is `(Ljava/lang/String;Ljava/lang/String;II)V`

[Reference](https://stackoverflow.com/questions/22038466/jni-signature-for-method)